org.eclipse.xtext.xbase.testing/build.gradle

```diff
@@ -11,3 +11,4 @@ dependencies {
 }
 
 sourceSets.main.java.srcDir 'testdata'
+sourceSets.main.java.srcDir 'deprecated'
```

Comment:
build.properties?

Reply:
this bundle does not even have a build.properties